src/screens/OtherProfile/index.js

```diff
@@ -229,7 +230,7 @@ const OtherProfile = () => {
         <View style={styles.wrapFollower}>
           <TouchableOpacity onPress={handleOpenFollowerUser} style={styles.wrapRow}>
             <React.Fragment>
-              <Text style={styles.textTotal}>{dataMain.follower_symbol}</Text>
+              <Text style={styles.textTotal(dynamicColor)}>{dataMain.follower_symbol}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `textTotal` style now accepts `dynamicColor` as an argument. Ensure that `dynamicColor` is correctly defined and passed where this style is used.
```diff
- <Text style={styles.textTotal}>{dataMain.follower_symbol}</Text>
+ <Text style={styles.textTotal(dynamicColor)}>{dataMain.follower_symbol}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->